pkg/loader/compose/v1v2.go

```diff
@@ -277,3 +323,88 @@ func checkLabelsPorts(noOfPort int, labels string, svcName string) error {
 	}
 	return nil
 }
+
+//returns true if duplicate volumes are there
+//it's possible when `volumes_from` key is used
+func findDuplicateVol(vols kobject.Volumes, volumes []kobject.Volumes) bool {
+	for _, v := range volumes {
+		if vols.MountPath == v.MountPath {
+			return true
+		}
+	}
+	return false
+}
+
+//returns all volumes associated with service, if `volumes_from` key is used, we have to retrieve volumes from the services which are mentioned there. Hence, recursive function is used here.
+func handleVolume(svcName string, komposeObject kobject.KomposeObject) (volume []kobject.Volumes) {
+	//if volumes-from key is present
+	if komposeObject.ServiceConfigs[svcName].VolumesFrom != nil {
+		//iterating over services from `volumes-from`
+		for _, depSvc := range komposeObject.ServiceConfigs[svcName].VolumesFrom {
+			//recursive call for retrieving volumes of services from `volumes-from`
+			dVols := handleVolume(depSvc, komposeObject)
+			var cVols []kobject.Volumes
+			//creating array of volumes of current service(svcName)
```

Comment:
A lot of your comments don't have the spacing after the `//`. 

Please add some new-lines too so that the separate sections are spaced out a bit better.